test/unit/extract/csv_test.py

```diff
@@ -55,12 +55,17 @@ def test_load_source(mock_pd):
 
 
 def test_extract():
-    extractor = FakeExtractor()
+    class SuperFakeExtractor(FakeExtractor):
+        def process_raw(self, df, page, **partition):
+            return df
```

Comment:
i removed the rename here. i think it would be easier overall if this test could either be ripped from the 176 extractor which would require all of the metadata bits necessary to really make an test extractor run.